spec/jobs/vre/create_ch31_submissions_report_job_spec.rb

```diff
@@ -53,6 +64,14 @@
       it 'sparks mailer with claims sorted by Regional Office' do
         subject
       end
+
+      it 'does not send if FeatureFlipper.staging_email?  is true' do
+        RSpec::Mocks.space.proxy_for(Ch31SubmissionsReportMailer).reset
+        RSpec::Mocks.space.proxy_for(FeatureFlipper).reset
+        expect(FeatureFlipper).to receive(:staging_email?).once.and_return(true)
+        expect(Ch31SubmissionsReportMailer).not_to receive(:build).with(submitted_claims)
+        subject
+      end
```

Comment:
@RudyOnRails Is this test the same as the one you added around line 50?

Reply:
@alsleaders It is similar but just within a different context.